server/services/conditionService.ts

```diff
@@ -67,17 +73,17 @@ export default class ConditionService {
     })
   }
 
+  async getAdditionalConditions(version: string): Promise<AdditionalConditionsConfig> {
+    const policy = await this.getConditions(version)
+    return this.parseResponse(policy.additionalConditions)
+  }
+
   async getAdditionalConditionByCode(
     searchCode: string,
     version: string = null
   ): Promise<AdditionalConditionAp | AdditionalConditionPss> {
```

Comment:
meh.. you could add a `typealias` for `AdditionalConditionAp | AdditionalConditionPss` as its used in a few places